### DISCUSSION TITLE
Other Branches

### DISCUSSION THREAD

**Discussion by pujanshah3333** [renovatebot/renovate#13107] [Request Help]
Hi,

We are running renovate in self hosted mode and in some repos we noticed that dashboard shows "Other branches" section and usually there is one or two branches/tasks. When we check for branches, there are no branch is GitHub repo with renovate, when we tick check box to convert them to PR after renovate run nothing changes except tick marks are gone. We ran renovate on a repo in debug mode and I'm attaching relevant part of the logs here. Can someone tell me if this is a bug in renovate or misconfiguration? Either way a way to fix this would be very helpful.

Thanks
```
DEBUG: Setting current branch to main (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: latest commit (repository=org_id/repo_name, branch=renovate/terraform-terraform)
       "branchName": "main",
       "latestCommitDate": "2021-12-14T08:06:44+00:00"
DEBUG: getBranchPr(renovate/terraform-terraform) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: findPr(renovate/terraform-terraform, undefined, open) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: findPr(renovate/terraform-terraform, undefined, closed) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: branchExists=false (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: dependencyDashboardCheck=undefined (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: recreateClosed is true (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Checking schedule(null, null) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: No schedule defined (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Branch needs creating (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Using reuseExistingBranch: false (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: manager.getUpdatedPackageFiles() reuseExistinbranch=false (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: terraform.updateArtifacts(terraform/20_resources/versions.tf) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: terraform.updateArtifacts(terraform/20_resources/main.tf) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: terraform.updateArtifacts(terraform/10_iam_roles/versions.tf) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: terraform.updateArtifacts(terraform/10_iam_roles/main.tf) (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: No package files need updating (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Getting updated lock files (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Writing package.json files (repository=org_id/repo_name, branch=renovate/terraform-terraform)
       "packageFiles": ["package.json"]
DEBUG: Ensuring package-lock.json is removed (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Writing any updated package files (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: No updated lock files in branch (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: No files to commit (repository=org_id/repo_name, branch=renovate/terraform-terraform)
DEBUG: Ensuring Dependency Dashboard (repository=org_id/repo_name)
 INFO: Blocked PR (repository=org_id/repo_name)
       "prBlockedBy": undefined,
       "prNo": undefined,
       "result": "no-work"
 INFO: Blocked PR (repository=org_id/repo_name)
       "prBlockedBy": undefined,
       "prNo": undefined,
       "result": "no-work"
```

**Comment by rarkins** [comment]
Seems it's a branch that Renovate thinks needs updating but is unable to. If you can reproduce it in a public repo on github.com then we can take a further look

**Reply by pujanshah3333** [reply]
Would showing my config file help? or you need a reproducible example?

**Reply by rarkins** [reply]
Needs to be an example

**Reply by pujanshah3333** [reply]
Can you take a look at this issue and repo and renovate runs in GitHub actions?
https://github.com/pujanshah3333/gradle-sample-app/issues/1

**Reply by rarkins** [reply]
Please post the debug log to a gist

**Reply by pujanshah3333** [reply]
Here are logs from last renovate run
https://gist.github.com/pujanshah3333/8cadbae556f7d4fa25226b24ba0504fa

**Reply by rarkins** [reply]
The time stamps make it too difficult to read, can you fix that?

**Reply by pujanshah3333** [reply]
I have updated gist

**Reply by rarkins** [reply]
It's possibly a lock file maintenance branch caught up in your package rule for Terraform. It may go away if that rule goes away, or if it is narrowed to not include lock file maintenance updates.

Can you reduce it to a minimal reproduction? https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md

**Reply by pujanshah3333** [reply]
Thanks, your comment pointed me in the right direction. I have removed lockFileMaintenance section from config and adding "updateLockFiles: true" in terraform section this has resulted in desired behavior. Thanks for your help.

**Comment by allanlewis** [comment]
For me, I think the problem was that my `pyproject.toml` (I'm using Poetry) specified a package with the version restriction `"*"`. Pinning this package allowed Renovate to recognise the desired version and then update it.
